Ethereal Artefacts/Ethereal Artefacts/Views/LoginView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  LoginView.swift
+//  Ethereal Artefacts
+//
+//  Created by Albert Khurshudyan on 3.05.23.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @State private var userEmail: String = ""
+    @State private var password: String = ""
+    @State private var isEmailValid = true
+    @State private var showPassword = false
+    @State private var loginError: String?
+    @State private var isLoggedIn: Bool = false
+    
+    let validUserEmail = "user@example.com"
+    let validPassword = "112233"
+    
+    var body: some View {
+        NavigationStack {
+            ZStack {
+                VStack{
+                    VStack{
```

Comment:
Note: No reason to have so many nested `VStack`s. The first one should suffice here.